src/InvalidConfigProviderException.php

```diff
@@ -10,6 +10,19 @@
 
 class InvalidConfigProviderException extends RuntimeException
 {
+    /**
+     * @param string $provider
+     * @return self
+     */
+    public static function fromDuplicateProvider($provider)
```

Comment:
declare parameter and return types

Reply:
When I created `fromDuplicateProvider`, I followed the structure of the other two methods (`fromNamedProvider` and `fromUnsupportedType`).
Ok, I'll add parameter and return types to this method.

Reply:
A leftover from before the types were added, most likely.